website/blog/2021-08-06-version-065.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Announcing React Native 0.65
+author: Luna Wei
+authorTitle: Engineer at Facebook
+authorURL: https://github.com/lunaleaps
+authorImageURL: https://avatars.githubusercontent.com/u/1309636?s=400&v=4
+authorTwitter: lunaleaps
+tags: [announcement, release]
```

Comment:
We need to add `date` to overwrite the filename date, before the blog post publish. 😉

Reply:
So something like
```suggestion
tags: [announcement, release]
date: 2021-08-17
```
@simek?

Reply:
Yup, I have left the link to the Docusaurus docs in the comment earlier, (https://github.com/facebook/react-native-website/pull/2717#pullrequestreview-724593515), but in the docs they only mention the base format, however there are few other options, which can be seen in the test file:
* https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-plugin-content-blog/src/__tests__/blogFrontMatter.test.ts#L302-L317

Reply:
Gotcha, I just committed @kelset's suggestion (btw thank you for the suggestions on the blog post and changelog!). I think the change meets the docusaurus date format.